components/molecules/carousel/__tests__/stories.js

```diff
@@ -79,13 +77,37 @@ export const withArray = args => (
   />
 )
 
+export const withCustomNav = args => (
+  <BaseComponent
+    {...args}
+    showArrows
```

Comment:
Well done @mohsenParvar . Just one thing, I think now we can change all `arrow` words we used in Carousel component to `nav` eg: `showArrows => showNavs`. We can also change the name of `Arrow` components names and files.